features/support/env.rb

```diff
@@ -1,5 +1,6 @@
+require 'aruba'
 require 'aruba/cucumber'
-require 'aruba/in_process'
+require 'aruba/processes/in_process'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.